packages/backendv2/src/models/quiz_answer.ts

```diff
@@ -71,8 +71,15 @@ class QuizAnswer extends Model {
         .orderBy("created_at")
         .page(page, pageSize)
     ).results
+
+    const answersAmount = (
+      await this.query()
+        .where("quiz_id", quizId)
+        .andWhereNot("status", "deprecated")
+    ).length
```

Comment:
This loads all records into memory and then calculates the length. Using sql `count()` would be preferable here.